I can not for the life of me install triangle

I am actually trying to install glumpy, which depends on triangle. Both cannot be installed either via pip or source install, and because both spit out the same error, I am now here. I am using Windows 10, installing in an Anaconda env with Python 3.7. The error I get after cloning the repo, doing
```
cd triangle
python setup.py install
```
is
```
E:\-----\VisualStudio\VC\Tools\MSVC\14.15.26726\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -DVOID=void -DREAL=double -DNO_TIMER=1 -DTRILIBRARY=1 -DANSI_DECLARATORS=1 -Ic -IE:\*****\Anaconda3\include -IE:\*****\Anaconda3\include -IE:\-----\VisualStudio\VC\Tools\MSVC\14.15.26726\ATLMFC\include -IE:\-----\VisualStudio\VC\Tools\MSVC\14.15.26726\include "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17134.0\cppwinrt" /Tctriangle/core.c /Fobuild\temp.win-amd64-3.7\Release\triangle/core.obj
core.c
E:\-----\VisualStudio\VC\Tools\MSVC\14.15.26726\bin\HostX86\x64\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:E:\*****\Anaconda3\libs /LIBPATH:E:\*****\Anaconda3\PCbuild\amd64 /LIBPATH:E:\-----\VisualStudio\VC\Tools\MSVC\14.15.26726\ATLMFC\lib\x64 /LIBPATH:E:\-----\VisualStudio\VC\Tools\MSVC\14.15.26726\lib\x64 "/LIBPATH:C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\lib\um\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17134.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17134.0\um\x64" /EXPORT:PyInit_core build\temp.win-amd64-3.7\Release\c/triangle.obj build\temp.win-amd64-3.7\Release\triangle/core.obj /OUT:build\lib.win-amd64-3.7\triangle\core.cp37-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.7\Release\c\core.cp37-win_amd64.lib
LINK : fatal error LNK1104: cannot open file 'OLDNAMES.lib'
error: command 'E:\\-----\\VisualStudio\\VC\\Tools\\MSVC\\14.15.26726\\bin\\HostX86\\x64\\link.exe' failed with exit status 1104
```
I masked my filesystem structure with ----- and *****. Those should be valid paths.
I would be happy to give you more information, I just don't know what :)

The error line 
```
LINK : fatal error LNK1104: cannot open file 'OLDNAMES.lib'
```
indicates it is trying to to link against a file, which is not mentioned anywhere in the triangle source.

--- a/.github/workflows/wheels.yml
+++ b/.github/workflows/wheels.yml
@@ -1,0 +1,37 @@
+name: Build Wheels
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheel on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: actions/setup-python@v1
+      name: Install Python
+      with:
+        python-version: '3.7'
+
+    - name: Install cibuildwheel
+      run: |
+        python -m pip install cibuildwheel==1.3.0
+
+    - name: Install Visual C++ for Python 2.7
+      if: startsWith(matrix.os, 'windows')
+      run: |
+        choco install vcpython27 -f -y
+
+    - name: Build wheel
+      run: |
+        python -m cibuildwheel --output-dir wheelhouse
+
+    - uses: actions/upload-artifact@v1
+      with:
+        name: wheels
+        path: ./wheelhouse

--- a/.github/workflows/wheels.yml
+++ b/.github/workflows/wheels.yml
@@ -7,34 +7,39 @@ jobs:
     name: Build wheel on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     env:
+      # pypy (Python with a JIT) builds don't work due to numpy issue 
+      CIBW_SKIP: pp*
       CIBW_TEST_REQUIRES: pytest
       CIBW_TEST_COMMAND: "pytest {project}/tests"
     strategy:
       matrix:
         os: [ubuntu-18.04, windows-latest, macos-latest]
-
     steps:
     - uses: actions/checkout@v1
-
     - uses: actions/setup-python@v1
       name: Install Python
       with:
         python-version: '3.7'
-
     - name: Install cibuildwheel
       run: |
         python -m pip install cibuildwheel==1.3.0
-
     - name: Install Visual C++ for Python 2.7
       if: startsWith(matrix.os, 'windows')
       run: |
         choco install vcpython27 -f -y
-
     - name: Build wheel
       run: |
         python -m cibuildwheel --output-dir wheelhouse
-
     - uses: actions/upload-artifact@v1
       with:
         name: wheels
         path: ./wheelhouse
+    - name: Upload To PyPi
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      # TODO : remove `if false` statement after secrets are set in Github UI
+      if: false
+      run: |
+        pip install twine
+        twine upload ./wheelhouse/*

--- a/.github/workflows/wheels.yml
+++ b/.github/workflows/wheels.yml
@@ -3,13 +3,12 @@ name: Build Wheels
 on: [push, pull_request]
 
 jobs:
-  env:
-    CIBW_TEST_REQUIRES: pytest
-    CIBW_TEST_COMMAND: "pytest --doctest-module triangle/tri.py"
-
   build_wheels:
     name: Build wheel on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
+    env:
+      CIBW_TEST_REQUIRES: pytest
+      CIBW_TEST_COMMAND: "pytest --doctest-module triangle/tri.py"
     strategy:
       matrix:
         os: [ubuntu-18.04, windows-latest, macos-latest]

--- a/.github/workflows/wheels.yml
+++ b/.github/workflows/wheels.yml
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       CIBW_TEST_REQUIRES: pytest
-      CIBW_TEST_COMMAND: "pytest --doctest-module triangle/tri.py"
+      CIBW_TEST_COMMAND: "pytest --doctest-module {project}/triangle/tri.py"
     strategy:
       matrix:
         os: [ubuntu-18.04, windows-latest, macos-latest]

--- a/.github/workflows/wheels.yml
+++ b/.github/workflows/wheels.yml
@@ -3,6 +3,10 @@ name: Build Wheels
 on: [push, pull_request]
 
 jobs:
+  env:
+    CIBW_TEST_REQUIRES: pytest
+    CIBW_TEST_COMMAND: "pytest --doctest-module triangle/tri.py"
+
   build_wheels:
     name: Build wheel on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}

--- a/.github/workflows/wheels.yml
+++ b/.github/workflows/wheels.yml
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       CIBW_TEST_REQUIRES: pytest
-      CIBW_TEST_COMMAND: "pytest --doctest-module {project}/triangle/tri.py"
+      CIBW_TEST_COMMAND: "pytest {project}/tests"
     strategy:
       matrix:
         os: [ubuntu-18.04, windows-latest, macos-latest]

--- a/tests/test_triangle.py
+++ b/tests/test_triangle.py
@@ -1,0 +1,48 @@
+import numpy as np
+
+from triangle import (triangulate,
+                      convex_hull,
+                      voronoi,
+                      delaunay)
+
+
+def test_triangulate():
+
+    v = [[0, 0], [0, 1], [1, 1], [1, 0]]
+    t = triangulate({'vertices': v}, 'a0.2')
+    assert np.allclose(t['vertices'],
+                       [[0.0, 0.0], [0.0, 1.0], [1.0, 1.0], [1.0, 0.0], [0.5, 0.5], [0.0, 0.5], [0.5, 0.0], [1.0, 0.5], [0.5, 1.0]])
+    assert np.allclose(t['vertex_markers'],
+                       [[1], [1], [1], [1], [0], [1], [1], [1], [1]])
+    assert np.allclose(t['triangles'],
+                       [[7, 2, 4], [5, 0, 4], [4, 8, 1], [4, 1, 5], [4, 0, 6], [6, 3, 4], [4, 3, 7], [4, 2, 8]])
+
+
+def test_delaunay():
+    pts = [[0, 0], [0, 1], [0.5, 0.5], [1, 1], [1, 0]]
+    tri = delaunay(pts)
+    assert np.allclose(tri,
+                       [[1, 0, 2], [2, 4, 3], [4, 2, 0], [2, 3, 1]])
+
+
+def test_hull():
+
+    pts = [[0, 0], [0, 1], [1, 1], [1, 0]]
+    segments = convex_hull(pts)
+    assert np.allclose(segments,
+                       [[3, 0], [2, 3], [1, 2], [0, 1]])
+
+
+def test_voronoi():
+
+    pts = [[0, 0], [0, 1], [0.5, 0.5], [1, 1], [1, 0]]
+    points, edges, ray_origin, ray_direct = voronoi(pts)
+    assert np.allclose(points,
+                       [[0.0, 0.5], [1.0, 0.5], [0.5, 0.0], [0.5, 1.0]])
+    assert np.allclose(edges,
+                       [[0, 2], [0, 3], [1, 2], [1, 3]])
+
+    assert np.allclose(ray_origin,
+                       [0, 1, 2, 3])
+    assert np.allclose(ray_direct,
+                       [[-1.0, 0.0], [1.0, 0.0], [0.0, -1.0], [0.0, 1.0]])

--- a/tests/test_triangle.py
+++ b/tests/test_triangle.py
@@ -1,48 +1,63 @@
 import numpy as np
 
-from triangle import (triangulate,
-                      convex_hull,
-                      voronoi,
-                      delaunay)
+from triangle import triangulate, convex_hull, voronoi, delaunay
 
 
 def test_triangulate():
 
     v = [[0, 0], [0, 1], [1, 1], [1, 0]]
-    t = triangulate({'vertices': v}, 'a0.2')
-    assert np.allclose(t['vertices'],
-                       [[0.0, 0.0], [0.0, 1.0], [1.0, 1.0], [1.0, 0.0], [0.5, 0.5], [0.0, 0.5], [0.5, 0.0], [1.0, 0.5], [0.5, 1.0]])
-    assert np.allclose(t['vertex_markers'],
-                       [[1], [1], [1], [1], [0], [1], [1], [1], [1]])
-    assert np.allclose(t['triangles'],
-                       [[7, 2, 4], [5, 0, 4], [4, 8, 1], [4, 1, 5], [4, 0, 6], [6, 3, 4], [4, 3, 7], [4, 2, 8]])
+    t = triangulate({"vertices": v}, "a0.2")
+    assert np.allclose(
+        t["vertices"],
+        [
+            [0.0, 0.0],
+            [0.0, 1.0],
+            [1.0, 1.0],
+            [1.0, 0.0],
+            [0.5, 0.5],
+            [0.0, 0.5],
+            [0.5, 0.0],
+            [1.0, 0.5],
+            [0.5, 1.0],
+        ],
+    )
+    assert np.allclose(
+        t["vertex_markers"], [[1], [1], [1], [1], [0], [1], [1], [1], [1]]
+    )
+    assert np.allclose(
+        t["triangles"],
+        [
+            [7, 2, 4],
+            [5, 0, 4],
+            [4, 8, 1],
+            [4, 1, 5],
+            [4, 0, 6],
+            [6, 3, 4],
+            [4, 3, 7],
+            [4, 2, 8],
+        ],
+    )
 
 
 def test_delaunay():
     pts = [[0, 0], [0, 1], [0.5, 0.5], [1, 1], [1, 0]]
     tri = delaunay(pts)
-    assert np.allclose(tri,
-                       [[1, 0, 2], [2, 4, 3], [4, 2, 0], [2, 3, 1]])
+    assert np.allclose(tri, [[1, 0, 2], [2, 4, 3], [4, 2, 0], [2, 3, 1]])
 
 
 def test_hull():
 
     pts = [[0, 0], [0, 1], [1, 1], [1, 0]]
     segments = convex_hull(pts)
-    assert np.allclose(segments,
-                       [[3, 0], [2, 3], [1, 2], [0, 1]])
+    assert np.allclose(segments, [[3, 0], [2, 3], [1, 2], [0, 1]])
 
 
 def test_voronoi():
 
     pts = [[0, 0], [0, 1], [0.5, 0.5], [1, 1], [1, 0]]
     points, edges, ray_origin, ray_direct = voronoi(pts)
-    assert np.allclose(points,
-                       [[0.0, 0.5], [1.0, 0.5], [0.5, 0.0], [0.5, 1.0]])
-    assert np.allclose(edges,
-                       [[0, 2], [0, 3], [1, 2], [1, 3]])
+    assert np.allclose(points, [[0.0, 0.5], [1.0, 0.5], [0.5, 0.0], [0.5, 1.0]])
+    assert np.allclose(edges, [[0, 2], [0, 3], [1, 2], [1, 3]])
 
-    assert np.allclose(ray_origin,
-                       [0, 1, 2, 3])
-    assert np.allclose(ray_direct,
-                       [[-1.0, 0.0], [1.0, 0.0], [0.0, -1.0], [0.0, 1.0]])
+    assert np.allclose(ray_origin, [0, 1, 2, 3])
+    assert np.allclose(ray_direct, [[-1.0, 0.0], [1.0, 0.0], [0.0, -1.0], [0.0, 1.0]])

--- a/triangle/__init__.py
+++ b/triangle/__init__.py
@@ -1,13 +1,13 @@
-from triangle.version import (
+from .version import (
     __version__,
 )
-from triangle.data import (
+from .data import (
     loads, load, get_data, show_data,
 )
-from triangle.plot import (
+from .plot import (
     plot, comparev, compare,
 )
-from triangle.tri import (
+from .tri import (
     triangulate,
     convex_hull, delaunay, voronoi,
 )

--- a/triangle/tri.py
+++ b/triangle/tri.py
@@ -1,4 +1,4 @@
-from triangle.core import triang
+from .core import triang
 
 terms = (
     ('pointlist', 'vertices'),

